kahuna/public/js/components/gr-top-bar/gr-top-bar.js

```diff
@@ -22,8 +22,11 @@ topBar.directive('grTopBarNav', [function() {
         transclude: true,
         // Annoying to have to hardcode root route here, but only
         // way I found to clear $stateParams from uiRouter...
-        template: `<a href="/search" class="home-link" title="Home">Home</a>
-                   <ng:transclude></ng:transclude>`
+        template: `<a href="/search" class="home-link" title="Home" ng:if="!hideHomeLink">Home</a>
```

Comment:
Hide the Home link. Thought there may be a way to create a specialised top-bar-with-home composed from a standard vanilla top-bar, rather than having toggles on the top-bar. Doesn't really matter tbh.